kafka/0.5/Dockerfile

```diff
@@ -35,11 +43,9 @@ RUN echo "$MD5HASH  /tmp/kafka.tgz" | md5sum -c - &&\
 COPY ./log4j.properties $KAFKA_HOME/config/log4j.properties
 
 #
-# Change ownership and switch user
+# Allow random UID to use Kafka
 #
-RUN chown -R kafka $KAFKA_HOME && \
-    chgrp -R kafka $KAFKA_HOME
-USER kafka
+RUN chmod -R g+w,o+w $KAFKA_HOME
```

Comment:
Hum, how odd. In my case the `chown` failed because the user was `kafka` but the file owned by `root` (which seems expected as per the issues linked above). I could make it work by moving the switch to kafka user behind the `chown`. Not sure why this issue only arises for me, but it seems the safest thing to do.